webpack_config/webpack.rhl.js

```diff
@@ -0,0 +1,68 @@
+const merge = require('webpack-merge');
```

Comment:
Rather than using a separate merge package can we use our existing webpack config function?

Reply:
I would like to get rid of the imperative style of `makeConfig.js`. It's a collection of conditional statements which makes it hard to optimise a build.

I'm counting the need for at least 6 different builds.
- browser:dev
- browser:prod
- browser:test
- electron:dev
- electron:prod
- electron:test
And see that there are multiple others...
- freezer?
- derivation-checker?
- html?

`webpack-merge` allows us to use a composition pattern for configuration files. It enables us to remove the need for flags and condition checks by simply having a `webpack-common.js` file that we extend or override with a target specific configuration e.g. `webapack-electron-dev.js`.
The execution path becoming simple to read, we will be able to modify and optimise each one in a declarative manner.